### ISSUE TITLE
Error message for reported errors include the error location

### ISSUE THREAD

**Issue by jensljungblad** [googleapis/google-cloud-ruby#4356]
I'm not sure if this is intended behavior or not. If it is, I don't understand why :) 

When I look at the list of reported errors in GCP, I don't even see the error names. I just see a long file path to the file where the error happened. It's followed by the error name, but this part is usually truncated. This seems to be because of this code, in `error_event.rb`:

```ruby
def self.from_exception exception
  backtrace = exception.backtrace
  message = "#{exception.message} (#{exception.class})"

  if !backtrace.nil? && !backtrace.empty?
    error_location = backtrace.first

    message = "#{error_location}: #{message}\n\t" +
              backtrace.drop(1).join("\n\t")
    file_path, line_number, function_name = error_location.split ":"
    function_name = function_name.to_s[/`(.*)'/, 1]
  end

  new.tap do |e|
    e.message = message
    e.file_path = file_path
    e.line_number = line_number.to_i
    e.function_name = function_name
  end
end
```

Specifically:

```ruby
message = "#{error_location}: #{message}\n\t" +
          backtrace.drop(1).join("\n\t")
```

Is this intentional? I was looking at the official documentation page (https://cloud.google.com/error-reporting/docs/viewing-errors) and in the picture of the error report list, the errors are not prefixed by the error location.

**Comment by dazuma** [comment]
I think this was originally the intended formatting, but things changed and we never updated this code. Agreed that the message should come first, rather than the first element of the backtrace.

**Comment by dazuma** [comment]
Fixed by @alanhala in #4379 . Will release within a few days.
